Dockerfile

```diff
@@ -74,6 +75,7 @@ RUN apt-get update && apt-get install -y \
 	python-websocket \
 	ubuntu-zfs \
 	xfsprogs \
+	vim-common \
```

Comment:
Yes, its used during the build an provides features to embed the license of the project into the strings of the binary.